py/client/pydeephaven/_table_interface.py

```diff
@@ -25,10 +29,11 @@ class TableInterface(ABC):
     """
 
     @abstractmethod
-    def table_op_handler(self, table_op) -> Any:
+    def table_op_handler(self, table_op) -> Union[Table, Query]:
+
         ...
 
-    def drop_columns(self, cols: Union[str, List[str]]) -> Any:
+    def drop_columns(self, cols: Union[str, List[str]]) -> Union[Table, Query]:
```

Comment:
The union looks great, but it seems inconsistent with the return doc.  Does the doc need changing?  Same comment throughout the file.

Reply:
You are right. Technically Union[Table, Query] is correct but the Sphinx generated doc looks inconsistent for Table. Changed back to just Table.